immuni_analytics/models/data_retention_document.py

```diff
@@ -0,0 +1,44 @@
+#    Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+#    Please refer to the AUTHORS file for more information.
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU Affero General Public License for more details.
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+from datetime import datetime
+from typing import Any, Dict
+
+from bson import ObjectId
+from mongoengine import Document
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class DataRetentionDocument(Document):
```

Comment:
I'm not really sure about the name. Can we call it `ExpiringDocument` or something similar? DataRetention is basically what the db does 😄

Reply:
However, `Expiring` suggests the document would expire on its own, while it's not what happens. 😞 I agree the current name is not good either! Let's see whether we can come up with something on point. 🤔

Reply:
Could be something like `RetentionPolicyCompliantDocumen`

Reply:
I would rather call them `AnalyticsDocument` or similar, instead of linking the class name to the current reason we created this superclass 😄

Reply:
✅